docker-compose.yml

```diff
@@ -132,6 +132,7 @@ services:
       COMPONENTS_DEFAULT_BACKEND: 'grandchallenge.components.backends.docker.DockerExecutor'
       DEBUG: 'true'
       PYTHONDONTWRITEBYTECODE: 1
+      DJANGO_LIVE_TEST_SERVER_ADDRESS: 'localhost:9200'
```

Comment:
I think it's better to add this as a default when you use `getenv` in `settings.py`.

Reply:
Actually that does not work. The `live_server` then does not pick this up if it's an environment variable.

Reply:
OK then it is better to set it as an ENV var for the test docker container. There is also a problem here that the port may be occupied. Is there a way to get around hardcoding it?

Reply:
The reason we hardcoded it is because we need to know the port for the `SessionControlView`, which is constructed outside of the test and hence is independent from the live_server that is set up for the test from scratch every time.

Reply:
In that case why can't you use the `settings` fixture and pass it through that way?

Reply:
Ah! Hadn't thought of that. That should work.

Reply:
Hm I can't get that work. The settings fixture in the test is seperate from the settings that retrieve the variable from in the get_context_data function on the view.

Reply:
Ah never mind, got it to work!